### DISCUSSION TITLE
Run App in Windows Terminal from Task Scheduler

### DISCUSSION THREAD

**Discussion by myokeeh** [microsoft/terminal#12364] [Q&A] [answered]
I'm trying to run a console app at startup in Windows Terminal when the PC starts.

It currently works just by running "C:\bedrock-server\bedrock_server.exe" but it runs outside of Windows Terminal.

`wt C:\bedrock-server\bedrock_server.exe` doesn't work. I kind of expected it to work since `wt.exe` is enabled as an App execution alias on my Windows 11 machine.

`"C:\Program Files\WindowsApps\Microsoft.WindowsTerminal_1.11.3471.0_x64__8wekyb3d8bbwe\wt.exe" "C:\bedrock-server\bedrock_server.exe"` works but it's now tied to a specific version of Windows Terminal.

Launching the app by double-clicking or via a shortcut runs it in Windows Terminal, so I'm unsure why Task Scheduler launches it the "old" way.

Is there a way to have the app always run in Windows Terminal in Task Scheduler?

**Comment by myokeeh** [comment]
Is there a way to do this?

**Reply by lhecker** [reply, marked as answer]
You have to use an absolute path for `wt`. For instance:
```
C:\Users\lhecker\AppData\Local\Microsoft\WindowsApps\wt.exe
```

(Replace "lhecker" with your username.)

![image](https://github.com/user-attachments/assets/adadcd0f-7afb-4e63-9438-2dd6c5461672)

**Reply by myokeeh** [reply]
Thank you! I just looked at more of documentation and got it exactly the way I want it. https://github.com/MicrosoftDocs/terminal/blob/main/TerminalDocs/command-line-arguments.md

I added `--window 0` so tasks would run on the open window as tabs.
